c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
For me it's not correct because it's possible to have ``doc_v1['geometry']: null``. In that case ``'geometry' in doc_v1`` is ``True`` but ``doc_v1['geometry']['geom']`` will cause an error.

In addition the current indentation of 
```
                'geometry1': doc_v1['geometry']['geom']
                if 'geometry' in doc_v1 else None,
```
is confusing and very readable.

I recommend
```
                'geometry1': doc_v1['geometry']['geom']
                    if 'geometry' in doc_v1 and doc_v1['geometry'] else None,
```